src/subscription_manager/cli.py

```diff
@@ -80,7 +78,7 @@ def _get_usage(self):
         Usage format strips any leading 'usage' so
         do not include it
         """
-        return _("%%prog %s [OPTIONS]") % self.name
+        return _("%%(prog)s {name} [OPTIONS]").format(name=self.name)
```

Comment:
Same problem here. It can be tested with this:

```
[root@thinkpad-t580 subscription-manager]# PYTHONPATH=./src python -m subscription_manager.scripts.subscription_manager syspurpose --help
usage: %(prog)s syspurpose [OPTIONS]

Convenient module for managing all system purpose settings

optional arguments:
  -h, --help            show this help message and exit
  --proxy PROXY_URL     proxy URL in the form of proxy_hostname:proxy_port
  --proxyuser PROXY_USER
                        user for HTTP proxy with basic authentication
  --proxypassword PROXY_PASSWORD
                        password for HTTP proxy with basic authentication
  --noproxy NO_PROXY    host suffixes that should bypass HTTP proxy
  --show                show current system purpose
```